app/models/miq_report.rb

```diff
@@ -284,10 +284,10 @@ def col_format_hash
   end
 
   def format_row(row, allowed_columns = nil, expand_value_format = nil)
-    @tz ||= get_time_zone(Time.zone)
+    tz = User.current_user.settings.fetch_path(:display, :timezone).presence || get_time_zone(Time.zone)
```

Comment:
it looks like that you are changing a behaviour here.
What do you think about this ?
`get_time_zone(User.current_user.settings.fetch_path(:display, :timezone).presence || Time.zone)`

Reply:
Hey Yuri, do I need this change, or did you decide it wasn't necessary after all and I should stick with the user setting?

Reply:
@lpichler 👍 🥇

Reply:
@d-m-u to the best of my knowledge, time profile assigned only to metric related reports and for those reports original fix does change behavior - time profile would be ignored and user's timezone will apply. So, let's change...

Reply:
may be we should test above statement and document it, since it will looks strange when some reports will honor user's timezone and others will ignored it ...